doozerlib/cli/config_plashet.py

```diff
@@ -738,7 +738,14 @@ def for_assembly(config: SimpleNamespace, image: Optional[str], rhcos: bool, bre
     runtime.initialize(mode="both", clone_source=False, clone_distgits=False, prevent_cloning=True)
     if not runtime.assembly_basis_event:
         raise DoozerFatalError("for-assembly must be run for an assembly with a basis event")
-    logger.warning(f'Constraining rpm search to stream assembly due to assembly basis event {runtime.assembly_basis_event}')
+    if not el_version:
+        if image:
+            el_version = runtime.image_map[image].branch_el_target()
+        elif rhcos:
+            el_version = 8  # FIXME: Currently RHCOS is RHEL8 based. Revisit here when this changes
```

Comment:
Interesting.. seems like elX should not be part of the rhcos dependencies schema. I'll make a ticket to refactor this. After that, should can blindly include whatever is in the list.

Reply:
https://issues.redhat.com/browse/ART-3156

Reply:
Currently all dependencies (including group, image member, and rhcos) are following this format:
```yaml
dependencies:
   rpms:
   - elN: NVR
   - elN: NVR
```

Looks like only group dependencies need el version. What do you think if we change the schema for image member and rhcos dependencies to the way we specify "is"?
```
dependencies:
    rpms:
    - nvr: NVR

```

Reply:
However, it doesn't seem to me that ART-3156 will eventually eliminate the hardcoded version. It is needed to populate group dependencies. Another place I hardcode the RHCOS RHEL version is https://github.com/openshift/aos-cd-jobs/pull/2690/files#diff-166229d927199bef1ae96efc6b610a1924440315ff92ba7de0527fc2c83f3498R107, for the same reason.

Reply:
Or, maybe we can define the RHEL version of RHCOS in ocp-build-data?